src/redux/books/books.js

```diff
@@ -0,0 +1,59 @@
+const ADD_BOOK = 'bookStore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+
+const initialState = {
+  books: [
+    {
+      id: '7a8f2ffb-862e-4d59-b47e-4b5231c81dfe',
+      category: 'Non-fiction',
+      title: 'A Short History of Nearly Everything',
+      author: 'Bill Bryson',
+      chapter: {
+        current: 'How to build a universe',
+        progress: '0',
+        total: '6',
+      },
+    },
+    {
+      id: '3fa5cecb-d4d7-4168-ba06-6db5ae9aba8c',
+      category: 'Mystery',
+      title: 'Lorem Ipsum',
+      author: 'Latin',
+      chapter: {
+        current: 'Dolor sit amet',
+        progress: '7',
+        total: '42',
+      },
+    },
+  ],
+};
```

Comment:
- [ ] It would be better if you set the initialstate as an empty array. Kindly fix this minor issue

![Capture](https://user-images.githubusercontent.com/64173189/139970598-c1e20c55-a909-4eed-a4d5-a9c923012638.JPG)